[thethingsnetwork] Update the to v3 (#31930)
home-assistant/home-assistant.github.io: source/_integrations/thethingsnetwork.markdown

The **Application ID** is used to identify the scope of your data.

![Application overview](/images/integrations/thethingsnetwork/applications.png)

> Enable the storage integration under the integrations menu:

Reviewer: ```suggestion
1. Under the integrations menu, enable the storage integration:
```

![Storage Integration](/images/integrations/thethingsnetwork/storage_integration.png)

> Ensure you have an [Uplink Payload Formatter](https://www.thethingsindustries.com/docs/integrations/payload-formatters/) for your device.

Reviewer: ```suggestion
2. Ensure you have an [Uplink Payload Formatter](https://www.thethingsindustries.com/docs/integrations/payload-formatters/) for your device.
```

![Payload Formatters](/images/integrations/thethingsnetwork/payload_formatters.png)

> You need an API key to be able to read the data from your application. The minimum rights is `Read Application Traffic (uplink and downlink)`.

Reviewer: ```suggestion
1. You need an API key to be able to read the data from your application. 
  - The minimum rights is `Read Application Traffic (uplink and downlink)`.
```

The **Application ID** is used to identify the scope of your data.

![Application overview](/images/integrations/thethingsnetwork/applications.png)

1. Under the integrations menu, enable the storage integration:

![Storage Integration](/images/integrations/thethingsnetwork/storage_integration.png)

2. Ensure you have an [Uplink Payload Formatter](https://www.thethingsindustries.com/docs/integrations/payload-formatters/) for your device.

![Payload Formatters](/images/integrations/thethingsnetwork/payload_formatters.png)

> 1. You need an API key to be able to read the data from your application. 

Reviewer: ```suggestion
3. You need an API key to be able to read the data from your application. 
```

>   - The minimum rights is `Read Application Traffic (uplink and downlink)`.

Reviewer: ```suggestion
  - The minimal required right is `Read Application Traffic (uplink and downlink)`.
```
